package.json

```diff
@@ -21,7 +21,7 @@
     "lint:all": "yarn lint && yarn lint:style",
     "fix:all": "yarn lint:fix && yarn fix:style",
     "doc:start": "lerna run dev --scope=docs",
-    "doc:build": "lerna run build --verbose --scope=docs && NODE_OPTIONS=--openssl-legacy-provider yarn build-storybook --docs -o ./website/.next/storybook",
```

Comment:
This isn't needed anymore? so would I remove cross-env?

Reply:
indeed by switching to webpack 5 it is no longer necessary